tests/e2e/autoscaling/autoscaler.go

```diff
@@ -108,53 +121,117 @@ var _ = Describe("Cluster size autoscaler [Serial][Slow]", func() {
 	})
 
 	It("should scale up or down if deployment replicas leave nodes busy or idle [Feature:Autoscaling]", func() {
```

Comment:
Is this case still needed?

Reply:
I could delete it since it's already covered in the second test case.

Reply:
let's keep it as is since it could help to identify issues when there're something wrong happened